[2025-08-29 19:21:35,946] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-19T13:04:41.720997+00:00 [queued]>
[2025-08-29 19:21:35,966] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-19T13:04:41.720997+00:00 [queued]>
[2025-08-29 19:21:35,966] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-29 19:21:35,966] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2025-08-29 19:21:35,966] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-29 19:21:35,996] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): fetch_and_send_weather> on 2025-08-19 13:04:41.720997+00:00
[2025-08-29 19:21:36,002] {standard_task_runner.py:52} INFO - Started process 444 to run task
[2025-08-29 19:21:36,008] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'fetch_and_send_weather', 'scheduled__2025-08-19T13:04:41.720997+00:00', '--job-id', '357', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmplmo6uhtv', '--error-file', '/tmp/tmp0caf1qe6']
[2025-08-29 19:21:36,009] {standard_task_runner.py:80} INFO - Job 357: Subtask fetch_and_send_weather
[2025-08-29 19:21:36,041] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-29 19:21:36,120] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-19T13:04:41.720997+00:00 [running]> on host a4b544519f2b
[2025-08-29 19:21:36,285] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=fetch_and_send_weather
AIRFLOW_CTX_EXECUTION_DATE=2025-08-19T13:04:41.720997+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-19T13:04:41.720997+00:00
[2025-08-29 19:21:36,287] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-29 19:21:36,289] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/scripts && python3 kafka/producer.py']
[2025-08-29 19:21:36,318] {subprocess.py:85} INFO - Output:
[2025-08-29 19:21:40,908] {subprocess.py:92} INFO - Envoi vers Kafka : {'temperature': 28, 'humidity': 74, 'windSpeed': 3.6, 'cloudCover': 98, 'rainIntensity': 0, 'timestamp': '2025-08-29T19:21:00Z'}
[2025-08-29 19:21:41,057] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-29 19:21:41,136] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=fetch_and_send_weather, execution_date=20250819T130441, start_date=20250829T192135, end_date=20250829T192141
[2025-08-29 19:21:41,182] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-29 19:21:41,257] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
